CMakeLists.txt

```diff
@@ -2257,6 +2258,15 @@ if (NOT TG_OWT_USE_PROTOBUF)
     )
 endif()
 
+if (NOT TG_OWT_USE_PIPEWIRE)
+    remove_target_sources(tg_owt ${webrtc_loc}
+        modules/desktop_capture/linux/base_capturer_pipewire.cc
+        modules/desktop_capture/linux/base_capturer_pipewire.h
+        modules/desktop_capture/linux/egl_dmabuf.cc
+        modules/desktop_capture/linux/egl_dmabuf.h
```

Comment:
Avoiding the build of `modules/desktop_capture/linux/egl_dmabuf.cc` is exactly what I had to do to fix building on OpenBSD today.